### ISSUE TITLE
App can not run up

### ISSUE THREAD

**Issue by wangzhongxing93** [31Carlton7/elisha#4]
I want to run this app,but in [pubspec.yaml](https://github.com/31Carlton7/elisha/blob/master/pubspec.yaml) ,I can not found https://github.com/31Carlton7/canton_design_system.git this repository.

**Comment by 31Carlton7** [comment]
Fixed

**Comment by wangzhongxing93** [comment]
I see you had changed ui repository, but this app still can not run up. New repository isn't match 'canton_design_system'

**Comment by kairos67** [comment]
I got error.![image](https://user-images.githubusercontent.com/61530964/212530035-2b0dffab-1304-49a2-b11c-df90cb8b62c7.png)

**Comment by 31Carlton7** [comment]
I am a little busy as of now but will release a fix for this. This entire repository/app is getting an entire refresh, as well as getting separated into 2 apps with a brand new UI. It's going to be a large project but would appreciate if you would like to join.

**Comment by LiveRock** [comment]
Dependencies issue even I modified it as :

```
dependency_overrides:
 canton_ui:
  git:
   url: https://github.com/31Carlton7/canton_ui.git
```

**Comment by 31Carlton7** [comment]
Fixed in #7
